DICUI/Data/Enumerations.cs

```diff
@@ -127,11 +133,25 @@ public enum KnownSystem
         TomyKissSite,
         VideoCD,
 
+        MarkerOtherEnd,
+
         #endregion
 
-        Custom = -1
+        Custom = 0x0EADBEEF
```

Comment:
What is this value now? It seems pretty arbitrary.

Reply:
I'll change it to something less arbitrary, it was supposed to be `0xDEADBEEF` but since it's signed I couldn't use it.